src/main/java/zd/zero/waifu/motivator/plugin/onboarding/UserOnboarding.kt

```diff
@@ -17,6 +17,10 @@ object UserOnboarding {
                 WaifuMotivatorPluginState.getPluginState().version = newVersion
                 UpdateNotification.display(ProjectManager.getInstance().defaultProject, newVersion)
             }
+
+        if (WaifuMotivatorPluginState.getPluginState().userId.isEmpty()) {
+            WaifuMotivatorPluginState.getPluginState().userId = UUID.randomUUID().toString()
```

Comment:
I just added this so that way we can know how many users are effected by the issues being reported.